custom_components/unifiprotect/camera.py

```diff
@@ -178,123 +266,120 @@ def brand(self):
     @property
     def model(self):
         """Return the camera model."""
-        return self._model
+        return self.device.type
 
     @property
     def is_recording(self):
         """Return true if the device is recording."""
-        return bool(
-            self._device_data["recording_mode"] != "never"
-            and self._device_data["online"]
-        )
+        return self.device.is_connected and self.device.is_recording
 
     @property
     def extra_state_attributes(self):
         """Add additional Attributes to Camera."""
-        chime_enabled = False
-        if self._device_type == DEVICE_TYPE_DOORBELL:
-            if self._device_data["has_chime"]:
-                chime_enabled = self._device_data["chime_enabled"]
 
         return {
             **super().extra_state_attributes,
-            ATTR_UP_SINCE: self._device_data["up_since"],
-            ATTR_CAMERA_ID: self._device_id,
-            ATTR_CHIME_ENABLED: chime_enabled,
-            ATTR_CHIME_DURATION: self._device_data["chime_duration"],
-            ATTR_IS_DARK: self._device_data["is_dark"],
-            ATTR_MIC_SENSITIVITY: self._device_data["mic_volume"],
-            ATTR_PRIVACY_MODE: self._device_data["privacy_on"],
-            ATTR_WDR_VALUE: self._device_data["wdr"],
-            ATTR_ZOOM_POSITION: self._device_data["zoom_position"],
+            ATTR_UP_SINCE: to_js_time(self.device.up_since),
```

Comment:
All datetime attributes are required to be in utc isoformat.  Its prefered to make them sensors though since sensors now require utc datetime objects.